packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -68,17 +54,11 @@ export const gcGenerationOptionName = "gcGeneration";
 export const gcTestModeKey = "Fluid.GarbageCollection.GCTestMode";
 /** Config key to expire a session after a set period of time. Defaults to true. */
 export const runSessionExpiryKey = "Fluid.GarbageCollection.RunSessionExpiry";
-/** Config key to disable the tombstone feature, i.e., tombstone information is not read / written into summary. */
-export const disableTombstoneKey = "Fluid.GarbageCollection.DisableTombstone";
 /** Config key to override throwing an error when tombstone object is loaded (requested). */
 export const throwOnTombstoneLoadOverrideKey =
```

Comment:
Wondering why we are keeping this one.  Definitely leave it for this PR, but maybe it gets removed in a follow-up

Reply:
Rajat had the same question and I replied her - https://github.com/microsoft/FluidFramework/pull/21992#issuecomment-2248575481:

We should ideally remove Fluid.GarbageCollection.ThrowOnTombstoneLoadOverride too but I think it's okay to leave if for now while we are still observing tombstone errors as a metric for GC accuracy. If that spikes, our partners may ask us a way to disable tombstone.